components/log-in/component.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC, useState } from 'react';
+import cx from 'classnames';
+
+import Icon from 'components/icon';
+import DropdownSelect from 'components/select';
+
+export interface LogInProps {
+  user: string,
+}
+
+export const LogIn: FC<LogInProps> = ({
+  user,
```

Comment:
Once https://github.com/Vizzuality/green-energy-data-platform/pull/13 is merged, use `useMe` hook instead of passing the user as prop.